### PR TITLE
New script "verify-solution" to run both on Ubuntu and Windows

### PR DESCRIPTION
**Pull request by alfonsograziano**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -9,7 +9,10 @@ on:
 jobs:
   test:
     name: Lint and test
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
```
Comment (alfonsograziano) [inline .github/workflows/ci.yml:12]: @simoneb my knowledge of GH Actions is very limited. Is this the right approach to test both on Ubuntu and on Win?

File: src/a02-assertions/verifiers/handler.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { sum, sumAsync } from './index.verify.js'
 import handler from '../../../verify/handler.js'
-import '../test/index.test.js'
+import(`../test/${process.env.USE_SOLUTION ? 'solution' : 'index'}.test.js`)
```
Comment (alfonsograziano) [inline src/a02-assertions/verifiers/handler.js:3]: @simoneb do you feel this approach is valid? If so, I'll implement it in the other files as well. 

Running the `verify` script directly and adding a new env var for me sounds robust enough but I'm open to suggestions :)

Comment (simoneb) [reply src/a02-assertions/verifiers/handler.js:3]: if it works, why not

Comment (alfonsograziano) [reply src/a02-assertions/verifiers/handler.js:3]: I'll do something more generic to make it work also in the new docusaurus project

**Review by simoneb — APPROVED** [review]

**Pull request merged**